packages/sync-server/src/routes/SyncInitiatePullRoute.ts

```diff
@@ -0,0 +1,28 @@
+import { Request } from 'express';
+import { Route } from '@tupaia/server-boilerplate';
+import { SyncServerInitiatePullRequest } from '@tupaia/types';
+
+export type SyncInitiatePullRequest = Request<
+  SyncServerInitiatePullRequest.Params,
+  SyncServerInitiatePullRequest.ResBody,
+  SyncServerInitiatePullRequest.ReqBody,
+  SyncServerInitiatePullRequest.ReqQuery
+>;
+
+export class SyncInitiatePullRoute extends Route<SyncInitiatePullRequest> {
+  public async buildResponse() {
+    const { ctx } = this.req;
+    const { sessionId } = this.req.params;
+    const { since: sinceString, projectIds, deviceId } = this.req.body;
+    const since = parseInt(sinceString, 10);
+    if (isNaN(since)) {
+      throw new Error('Must provide "since" when creating a pull filter, even if it is 0');
+    }
```

Comment:
The overlap with this and the client route gets me thinking, though it may be one of those cases where it's better to just RY